activerecord/lib/active_record/connection_adapters/postgresql/schema_definitions.rb

```diff
@@ -45,130 +45,780 @@ def primary_key(name, type = :primary_key, **options)
+        #
+        #   # db/migrate/20131009135255_create_profiles.rb
+        #   class CreateProfiles < ActiveRecord::Migration[8.1]
+        #     def change
+        #       enable_extension "hstore" unless extension_enabled?("hstore")
+        #
+        #       create_table :profiles do |t|
+        #         t.hstore :settings
+        #       end
+        #     end
+        #   end
+        #
+        #   # app/models/profile.rb
+        #   class Profile < ApplicationRecord
+        #   end
+        #
+        #   Profile.create(settings: { "color" => "blue", "resolution" => "800x600" })
+        #
+        #   irb> profile = Profile.first
+        #   irb> profile.settings
+        #   # => {"color"=>"blue", "resolution"=>"800x600"}
+        #
+        #   irb> profile.settings = { "color" => "yellow", "resolution" => "1280x1024" }
+        #   irb> profile.save!
+        #
+        #   irb> Profile.where("settings -> 'color' = ?", "yellow")
+        #
+        # 🔗 See also: {PostgreSQL type definition}[https://www.postgresql.org/docs/current/static/hstore.html] and {Hstore functions and operators}[https://www.postgresql.org/docs/current/static/hstore.html#id-1.11.7.26.5]
+        #
+        # 💡 When to use +hstore+ vs +jsonb+?
+        #
+        # Use +hstore+ when you need a simple, flat key-value store where both keys and values are strings.
+        # It’s lightweight and efficient for storing unstructured settings or metadata. If you require nested
+        # structures, typed values (e.g. numbers, booleans), or advanced querying capabilities, consider using
+        # +jsonb+ instead.
 
         ##
         # :method: inet
         # :call-seq: inet(*names, **options)
+        #
+        # Adds an +inet+ column for storing IPv4 or IPv6 addresses with or without subnet masks.
+        # In Ruby, this is mapped to an {IPAddr}[https://docs.ruby-lang.org/en/master/IPAddr.html] object.
+        #
+        # Example:
+        #
+        #   create_table(:devices, force: true) do |t|
+        #     t.inet "ip_address"
+        #   end
+        #
+        # You can assign and query values as strings:
+        #
+        #   irb> device = Device.create(ip_address: "192.168.1.1")
+        #   irb> device.ip_address.class
+        #   # => IPAddr
+        #
+        #   irb> device.ip_address
+        #   => #<IPAddr: IPv4:192.168.1.12/255.255.255.255>
 
         ##
         # :method: interval
         # :call-seq: interval(*names, **options)
+        #
+        # Adds an +interval+ column for storing durations of time. It represents
+        # elapsed time (e.g., hours, days, months). In Ruby, this maps to
+        # {ActiveSupport::Duration}[https://api.rubyonrails.org/classes/ActiveSupport/Duration.html]
+        # objects.
+        #
+        # Example:
+        #
+        #   create_table :events do |t|
+        #     t.interval "duration"
+        #   end
+        #
+        # You can assign durations using ActiveSupport helpers:
```

Comment:
```suggestion
        # Durations can be assigned using ActiveSupport helpers:
```
Or maybe the shorter/clearer version:
```suggestion
        # Assign durations using ActiveSupport helpers:
```